src/Logo - Google- Monochrome.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
File name is missing a space after "Google"

Reply:
Fixed that here and in Figma. Nice catch!

Just to document: Since we strip the spaces there is no name difference for the generated version

Reply:
yup, but at least the source files are kept neat